app/services/applications/revert_to_pending.rb

```diff
@@ -0,0 +1,43 @@
+module Applications
+  class RevertToPending
+    REMOVEABLE_DECLARATION_STATES = %w[submitted voided ineligible].freeze
+
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+
+    attr_reader :application
+
+    attribute :change_status_to_pending
+    delegate :lead_provider_approval_status, to: :application
+
+    validates :change_status_to_pending, inclusion: { in: %w[yes] }
+    validates :lead_provider_approval_status, inclusion: { in: %w[accepted] }
```

Comment:
when going through the form now, if I select nothing I only expect to see the prompt for yes/no however we're getting this validation as well

Reply:
not sure I understand this validation here, as it triggers regardless if I select yes or no 🤔, do we need it?

Reply:
ah we also allow rejected as well not just accepted for reverting

Reply:
I did a change around this after the other reviews but before yours, it might be I've got something wrong on the 'no selection' scenario, I'll retest